2021-11-22 15:57:42,662 INFO    MainThread:159945 [wandb_setup.py:_flush():71] setting env: {'project': '11_21train'}
2021-11-22 15:57:42,662 INFO    MainThread:159945 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-22 15:57:42,662 INFO    MainThread:159945 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211122_155742-21ufc54a/logs/debug.log
2021-11-22 15:57:42,662 INFO    MainThread:159945 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211122_155742-21ufc54a/logs/debug-internal.log
2021-11-22 15:57:42,663 INFO    MainThread:159945 [wandb_init.py:init():404] calling init triggers
2021-11-22 15:57:42,663 INFO    MainThread:159945 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-22 15:57:42,663 INFO    MainThread:159945 [wandb_init.py:init():449] starting backend
2021-11-22 15:57:42,664 INFO    MainThread:159945 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-22 15:57:42,682 INFO    MainThread:159945 [backend.py:ensure_launched():199] starting backend process...
2021-11-22 15:57:42,693 INFO    MainThread:159945 [backend.py:ensure_launched():204] started backend process with pid: 160005
2021-11-22 15:57:42,694 INFO    MainThread:159945 [wandb_init.py:init():458] backend started and connected
2021-11-22 15:57:42,696 INFO    MainThread:159945 [wandb_init.py:init():519] updated telemetry
2021-11-22 15:57:42,718 INFO    MainThread:159945 [wandb_init.py:init():549] communicating current version
2021-11-22 15:57:43,824 INFO    MainThread:159945 [wandb_init.py:init():554] got version response 
2021-11-22 15:57:43,825 INFO    MainThread:159945 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-22 15:57:44,204 INFO    MainThread:159945 [wandb_init.py:init():592] starting run threads in backend
2021-11-22 15:57:44,252 INFO    MainThread:159945 [wandb_run.py:_console_start():1816] atexit reg
2021-11-22 15:57:44,253 INFO    MainThread:159945 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-22 15:57:44,253 INFO    MainThread:159945 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-22 15:57:44,256 INFO    MainThread:159945 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-22 15:57:44,256 INFO    MainThread:159945 [wandb_init.py:init():619] run started, returning control to user process
2021-11-22 15:57:44,264 INFO    MainThread:159945 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': '11_22baseline_network=resnext,cardinality=16', 'no': '0', 'model': 'resnext', 'print_freq': 10, 'layers': 32, 'droprate': 0.3, 'cardinality': 16, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00052_network=resnext,cardinality=16', 'commit': 'not_set'}
2021-11-22 15:58:17,574 INFO    MainThread:159945 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-22 15:58:17,574 INFO    MainThread:159945 [wandb_run.py:_restore():1758] restore
2021-11-22 15:58:19,694 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 976
}

2021-11-22 15:58:19,992 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 976
}

2021-11-22 15:58:20,365 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 976
}

2021-11-22 15:58:21,250 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 8315
}

2021-11-22 15:58:21,353 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 8342
}

2021-11-22 15:58:21,455 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 8342
}

2021-11-22 15:58:21,558 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 8342
}

2021-11-22 15:58:21,661 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 976
  total_bytes: 8342
}

2021-11-22 15:58:21,764 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:21,866 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:21,969 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:22,071 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:22,174 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:22,277 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:22,628 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}

2021-11-22 15:58:23,430 INFO    MainThread:159945 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8342
  total_bytes: 8342
}
local_info {
}

2021-11-22 15:58:24,506 INFO    MainThread:159945 [wandb_run.py:_append_files():2184] logging synced files
